tests/integration/test_data/npm_smoketest_lockfile2/bom.json

```diff
@@ -0,0 +1,371 @@
+{
+  "bomFormat": "CycloneDX",
+  "components": [
+    {
+      "name": "bar",
+      "version": "1.0.0",
+      "type": "library"
+    },
+    {
+      "name": "baz",
+      "version": "1.0.0",
+      "type": "library"
+    },
```

Comment:
`baz` is only reported in lockfiles 2 and 3, not in lockfile 1

all 3 versions also report `not-baz`

both `baz` and `not-baz` refer to this package I believe: https://github.com/cachito-testing/npm-cachi2-smoketest/blob/lockfile-v1/baz/package.json

it should be reported only as `not-baz` (and then, later, the purl should report the file path)

Reply:
agreed, fixed